v2/azure-arm.yaml

```diff
@@ -515,6 +515,11 @@ status:
       basePath: 'securityinsights/resource-manager/common',
       namespace: 'Microsoft.SecurityInsights'
     },
+    {
```

Comment:
Does this change need to be rebased onto master now that the submodule update has been done?

Reply:
Yes. I've rebased and force pushed.